couchdbkit/client.py

```diff
@@ -104,6 +112,11 @@ def __init__(self, uri='http://127.0.0.1:5984',
         else:
             self.res = self.resource_class(uri, **client_opts)
         self._uuids = deque()
+        self.cloudant_client = CouchDB('', '', url=uri, admin_party=True, connect=True)
```

Comment:
I'm curious why use `admin_party=True`? Does this change the auth mode of the database to admin party?

Reply:
The url is passed in with the username/pass already in the uri. I think that was a temporary workaround since i'll need to change the way that's called